ci/integration.yml

```diff
@@ -7,13 +7,21 @@ image_resource:
     tag: 1.13-stretch
 inputs:
   - name: repo
-params:
-  ENABLE_INTEGRATION_TESTS: "true"
 run:
   path: sh
   args:
     - -e
     - -c
     - |
+      echo "configuring integration tests..."
+      AWS_ACCOUNT_ID=$(curl http://169.254.169.254/latest/dynamic/instance-identity/document \
```

Comment:
oof, okay, no aws sts get-caller-identity I guess because the image we have doesn't awscli etc.?

Reply:
yeah exactly ... bit hacky ... but works :)